feature-wallet-impl/src/main/java/io/novafoundation/nova/feature_wallet_impl/data/network/blockchain/assets/transfers/equilibrium/EquilibriumAssetTransfers.kt

```diff
@@ -55,6 +57,14 @@ class EquilibriumAssetTransfers(
         recipientCanAcceptTransfer(assetSourceRegistry)
     }
 
+    override suspend fun totalCanDropBelowMinimumBalance(chainAsset: Chain.Asset): Boolean {
+        return false
```

Comment:
I think it should be `true`for everything except `native`

Reply:
Yes, exactly. Mixed up the logic